benefits/core/migrations/local_fixtures.json

```diff
@@ -64,33 +64,33 @@
       "client_name": "benefits-medicaregov",
       "client_id": "319efaea-615b-4cd4-958f-e6cd2fd31646",
       "authority": "https://example.com",
-      "scheme": "dev-cal-itp_benefits"
+      "scheme": "bluebutton"
```

Comment:
I think part of the point of this was to not encode our _actual_ values here. You do a good job in other docs sections referencing the LastPass fixtures for Compiler devs, any other reason to make these update in light of that?

Reply:
My assumption was that we're happy to include anything that isn't secret(ish?). In this specific case, the previous value is the actual value we use in `dev` for the Login.gov config, so I assumed it wasn't a secret, and that this Medicare one not matching the value that's shown on `dev` was simply an oversight.

Let me know if I misinterpreted any of that, or if you were also talking about any other lines I changed in `local_fixtures.json`.